src/SFML/Audio/SoundFileReaderFlac.hpp

```diff
@@ -133,7 +133,6 @@ class SoundFileReaderFlac : public SoundFileReader
     ////////////////////////////////////////////////////////////
     FLAC__StreamDecoder* m_decoder;      ///< FLAC decoder
     ClientData           m_clientData;   ///< Structure passed to the decoder callbacks
```

Comment:
Now that m_channelCount is gone, you can remove two spaces before the comments XD